workshops/scrapbook_css/README.md

```diff
@@ -0,0 +1,249 @@
+  
+  --fonts-body: 'Font name', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI';
+
+}
+```
+
+
+Replace ` ‘Font name’` with the name of your font, just keep the quotes! We’ve left the other fonts there as a backup for if the font we want won’t load.
+
+I went techno style, with the [Audiowide](https://fonts.google.com/specimen/Audiowide?category=Display&sidebar.open=true&selection.family=Audiowide) font.
+
+<img src="https://cloud-loib1whnd.vercel.app/screenshot_2020-09-12_at_8.01.02_pm.png" width="380" alt="Green and purple themed Scrapbook">
+
+## **Crazy CSS Animations - Part 1 ✌︎**
+
+We’ve got a fancy font and some magical colours but this site still needs some jazzing up! Looks animate it!
+First off, we’re going to make our text rainbow animated.
+
+Animations are made with keyframes, where you set the state of the object (in CSS properties) at different stages of the animation. If you've got any experience with video animation keyframing works very similarly here!
+
+Let’s define that animation:
+
+```css
+@keyframes rainbow {
+  14% {
+    color: #ff0000;
+  }
+  28% {
+    color: #ffa500;
+  }
+  42% {
+    color: #ffff00;
+  }
+  56% {
+    color: #008000;
+  }
+  70% {
+    color: #0000ff;
+  }
+  84% {
+    color: #4b0082;
+  }
+  100% {
+    color: #ee82ee;
+  }
+}
+```
+
+What this does is, 14% through the animation it will be #ff0000 (red), 28% through the animation it will be #ffa500 (orange) and so on…
+
+Now, we need to tell the web browser which element should have the animation applied to it. How we identify an element is with a class name and a `.` before it. To start we want to type this:
+
+```css
+.header-title-name{
+
+}
+```
+
+In between the two curly braces, we can type stuff that changes what all the elements with the class: header-title-name have. Inside of this, let’s ask the web browser to add our animation to the element. We can do this by adding this:
+
+```css
+animation: rainbow 5s infinite;
+```
+
+We want to add this inside the curly braces. What this does is apply the rainbow animation and run it for an infinite amount of times for 5 seconds each.
+
+Here's what I got:
+
+<img src="https://cloud-1fmtzoja5.vercel.app/ezgif-5-db525cfe2a47.gif" width="380" alt="Rainbow animated text">
+
+I’ve got a challenge for you. Let’s make that profile picture spin! 
+
+Here are a few things you’ll need to know:
+
+- The class for the profile picture is `header-title-avatar`
+- To rotate a image use: `transform: rotate(20deg)`
+
+So can you do it? The solution is [here](#spinner-solution).
+
+<img src="https://cloud-ojh0xf17r.vercel.app/ezgif-5-5540a1713ebc.gif" width="380" alt="Spinning profile picture">
+
+## **Crazy CSS Animations - Part 2** **✌︎**
+
+Animations can also happen when you hover over an element as well. This is through pseudo-classes, a word that you can add to your selector to specify a specific state. For example, `.header-title-avatar:hover` applies when the avatar is being hovered over.
```

Comment:
Probably want a space in the CSS here

Reply:
ahh I watched the video, this probs can be resolved lol